CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 ### Added
 
 - We added a field showing the BibTeX/biblatex source for added and deleted entries in the "External Changes Resolver" dialog. [#9509](https://github.com/JabRef/jabref/issues/9509)
+- We added user specific comment field so that user can make comments with username suffix.[#543](https://github.com/koppor/jabref/issues/543)
```

Comment:
`Space forgotten:

```suggestion
- We added user specific comment field so that user can make comments with username suffix. [#543](https://github.com/koppor/jabref/issues/543)
```